README.md

```diff
@@ -9,12 +9,79 @@
 
 <img src="assets/spectrum-css_illustration_desktop.png">
 
+## Quick start
```

Comment:
```suggestion
## Quick start guide

This example will show you how to quickly install Spectrum CSS and use it to display a component.
```
Thoughts on adjusting the name to "Quick start guide"?

I think it needs a small intro here to explain what this guide does; the text I've suggested could probably use some work and elaboration.